bin/image_size_inspector.py

```diff
@@ -0,0 +1,42 @@
+# Goal: This script is used to inspect the image size so we can figure out the best size to normalize at
+#       This can also be used to calculate the amount of images with mapfile
+# TODO: Make this script so that I can customize the resolution
+# Result: I have decided using this script to settle on a resolution of 256x256
+#         Verified that map file and original image have the same resolution
+
+# This only takes images with a mask file - with labeled data
+
+from PIL import Image
+import os
+
+def get_num_pixels(file_path):
+  width, height = Image.open(file_path).size
+  return (width, height)
+
+def analyze_directory(directory_path):
+  directory = os.fsencode(directory_path)
```

Comment:
Is the `os.fsencode` necessary with `os.listdir`? I think it is only needed for command line arguments